docs/en/createapps.md

```diff
@@ -21,15 +21,15 @@ https://usermanagement.adobe.io/v2/usermanagement/...
 
 **NOTE:** In syntax statements, this address is shortened to **[UM_Server]**.
 
-There are separate endpoints under the user-management server for write operations (managing the enterprise user base), and read operations (retrieving information from the enterprise user base). The API defines a set of specific write actions that you can use to create, update, and delete user accounts, and manage Adobe product access for users. The read operations retrieve information about users and product configurations in your organization.
+There are separate endpoints under the user-management server for write operations (managing the enterprise user base), and read operations (retrieving information from the enterprise user base). The API defines a set of specific write actions that you can use to create, update, and delete user accounts, and manage Adobe product access for users. The read operations retrieve information about users and product profiles in your organization.
 
 ***
 
 * [Authorization](#authorization)
 * [Manage your Adobe Users](#manage-your-adobe-users)
 * [Query your Adobe Users and Product Access](#query-your-adobe-users-and-product-access)
 
-For a Python code walkthrough and samples of actual API calls, see the [User Management Walkthrough](samples/index.md).
+For a Python code walkthrough and samples of actual API calls, see the [User Management Walkthrough](samples/index.html).
```

Comment:
Links to section shouldn't need the utl in parens.  Jekyll will automatically fill that in.   If you do need the parens, it should go to the .md file.  .html is incorrect as far as I know.

Reply:
I couldn't make link styles targeting the markdown file work with Jekyll. 

```
[User Management Walkthrough](samples/index.md)
```

This produces the following built:
```
Not Found

`/en/samples/index.md' not found.
```

The following works on our own git pages site here: https://git.corp.adobe.com/pages/JIL-v2/umapi-docs/en/createapps.html
```
[User Management Walkthrough](samples/index.html)
```